frontend/src/app/main/data/workspace/colors.cljs

```diff
@@ -390,3 +392,22 @@
     (update [_ state]
       (-> state
           (assoc-in [:workspace-global :editing-stop] spot)))))
+
+(defn manage-color-in-palette
+  [color is-alt?]
+  (ptk/reify ::manage-color-in-palette
+    ptk/WatchEvent
+    (watch [_ state _]
+      (let [objects  (wsh/lookup-page-objects state)
+            selected (->> (wsh/lookup-selected state)
+                          (cph/clean-loops objects))
+            ids      (into []
+                           (comp (keep (d/getf objects))
+                                 (mapcat (fn [shape]
+                                           (cons (:id shape)
```

Comment:
This causes that the fill or stroke is applied to the group shapes (it shouldn't, only to children).